research/conditional/train/configs/bert_base_exp=512_id=316.yaml

```diff
@@ -0,0 +1,45 @@
+# grid args
+runner: "research.conditional.train.cc_train"
+time: "5-05:00:00"
+n_gpus: 1
+cuda_visible: "7"
+
+# train params
+params:
+  # model
+  model_type: "bert"
+  dataset_type: "wikibook"
+  dmodel: 768
+  n_att_heads: 12
+  n_blocks: 12
+  cutoff: 256
+  batch_size: 64
+  loss_checkpoint_chungs: 4
```

Comment:
Czemu tylko 4 akurat w największym modelu? Wciąż dużo zużycia RAM będzie na loss layer wtedy, co nie?

Reply:
tunowałem tak: znajdź największy batch który się nie wywala z loss_checkpoint_chungs takim jak był w innych czyli 64, a potem znajdź najmniejsze chungsy przy których się nie wywala (bo im większe chungs tym wolniej działa i jest aktualnie duża różnica)

Reply:
czy możnaby to sprawdzić ponownie?